### DISCUSSION TITLE
Can't set a border-width by CSS var like border-[--my-css-var]

### DISCUSSION THREAD

**Discussion by hansipete** [tailwindlabs/tailwindcss#11988] [Help] [answered]
This will result in the `border-color` being set.

tailwindcss v3.3.1
vite/4.1.1 darwin-arm64 node-v20.3.1
Node.js v20.3.1

https://play.tailwindcss.com/h24E9Hgscb

**Comment by adamwathan** [comment, marked as answer]
Hey! This will fix it:

https://play.tailwindcss.com/L7L28GKVhL

Check out the documentation on resolving ambiguities in arbitrary values:

https://tailwindcss.com/docs/adding-custom-styles#resolving-ambiguities

**Reply by hansipete** [reply]
So I learned something new today :) Thank you (and sorry for filing it as a bug in the first place.)
